storage/postgres_store.go

```diff
@@ -0,0 +1,124 @@
+package storage
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/arhsxro/platform-go-challenge/config"
+	"github.com/arhsxro/platform-go-challenge/models"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+type PostgresStore struct {
+	db *sqlx.DB
+}
+
+func isValidAssetType(paramType string) bool {
```

Comment:
Nice that you added a validation. Can you think of a better a way that this could be handled?

Reply:
I could probably move the validation to my handler in order not to proceed more if there is a validation err.
Also I could use a map instead of slice to avoid the loop.